test_check_if_anagrams.py

```diff
@@ -1,6 +1,9 @@
 def are_anagrams(str1, str2):
   # Remove the `return None` code below and add your own code.
-  return None
+  if sum(bytearray(str1.upper())) == sum(bytearray(str2.upper())):
```

Comment:
Nice! Can you please walk me through what `bytearray()` returns -- maybe using some illustrative examples similar to this problem -- and why you chose to compare the `sum` of the two `bytearray(...)` values. Thanks!